README.md

```diff
@@ -40,7 +40,13 @@ $ chmod +x coveralls.phar
 To install php-coveralls with Composer, run the following command:
 
 ```sh
-$ composer require satooshi/php-coveralls --dev
+$ composer require satooshi/php-coveralls
```

Comment:
Bottom line is that it's not necessary and far more dangerous than using a version lock.